packages/flutter/lib/src/material/text_field.dart

```diff
@@ -800,6 +801,37 @@ class TextField extends StatefulWidget {
       decorationStyle: TextDecorationStyle.wavy,
   );
 
+  /// Default builder for the spell check suggestions toolbar in the Material
+  /// style.
+  ///
+  /// See also:
+  ///  * [SpellCheckConfiguration.spellCheckSuggestionsToolbarBuilder], the
+  //     builder configured to show a spell check suggestions toolbar.
+  static Widget defaultSpellCheckSuggestionsToolbarBuilder(
+    BuildContext context,
+    EditableTextState editableTextState,
+  ) {
+    final SuggestionSpan? spanAtCursorIndex =
+      findSuggestionSpanAtCursorIndex(
+        editableTextState.currentTextEditingValue.selection.baseOffset,
+        editableTextState.spellCheckResults!.suggestionSpans,
+      );
```

Comment:
It looks like spanAtCursorIndex could be calculated inside of buildButtonItems.  Would that be beneficial to the architecture at all?  It might be easier for users to just call buildButtonItems with `context` and `editableTextState` without having to worry about passing the suggestion span, since that could be confusing as to how to create that.